src/bilder/images/edxapp/plugins/git_export_import.py

```diff
@@ -1,28 +1,38 @@
 from pathlib import Path
 
+from pyinfra.api import deploy
 from pyinfra.operations import files, pip, ssh
 
-pip.packages(
-    name="Install edx-git-auto-export",
-    packages=[
-        "git+https://github.com/mitodl/edx-git-auto-export.git@v0.3#egg=edx-git-auto-export",  # noqa: E501
-    ],
-    present=True,
-    virtualenv="/edx/app/edxapp/venvs/edxapp/",
-    sudo_user="edxapp",
-)
 
-files.directory(
-    name="Create .ssh directory for www-data user to clone course repositories",
-    path=Path("/var/www/.ssh/"),
-    present=True,
-    mode="0700",
-    user="www-data",
-    group="www-data",
-)
-for host in ("github.com", "github.mit.edu"):
-    ssh.keyscan(
-        name=f"Add {host} public SSH fingerprint for course import/export",
-        hostname=host,
-        sudo_user="www-data",
+@deploy("Set up git auto export")
+def git_auto_export(state=None, host=None):
+    pip.packages(
+        name="Install edx-git-auto-export",
+        packages=[
```

Comment:
Can't this be part of the other pkgs?

Reply:
It _can_, but I set this up as its own operation because of the other steps that are necessary to make it work.